### ISSUE TITLE
Data Access

### ISSUE THREAD

**Issue by NewmoonLee** [OSU-CEOAS-Schmittner/UVicMOBI_fct_npzd_o2_n_fe_ca_caco3#1]
Where could I find other Uvic's ecosystem model data

**Comment by kaipak** [comment]
@andreasschmittner Can you help with this question?

**Comment by andreasschmittner** [comment]
What other data?

**Comment by NewmoonLee** [comment]
Could you please tell me where could I find the monthly Chlorophyll or Nitrient model data from UVic?

**Comment by andreasschmittner** [comment]
I can send them to you. What exactly do you want? MOBI does not calculate Chlorophyll as a prognostic variable. I think we've calculated it previously from phytoplankton biomass. MOBI calculates nitrate, phosphate, and iron as nutrients. Let me know which you want. Best if you write me an email. Github is not a place for model output.

**Comment by NewmoonLee** [comment]
I totally agree with you, and thank you so much! I found you email address on the website of Oregon State University, is that the one you use now? I will write you a email later if that's true.

**Comment by andreasschmittner** [comment]
My email address is aschmitt@coas.oregonstate.edu

**Comment by NewmoonLee** [comment]
I have already sent you a email, please check it when you have time ^_^ ~
Best Wishes to you !
